### ISSUE TITLE
Cyber Challenge

### ISSUE THREAD

**Issue by daweim0** [RockRaidersInc/ROS-Main#33]
What is it, How much work would it be, and are the points worth it? If so we should do it

**Comment by sigi73** [comment]
Yeah, its mostly here if a new member is more interested in security they could work on the cyber challenge.

**Comment by daweim0** [comment]
Prize money is prize money, let's be cash flow positive this year
